pkg/backends/locust/backend.go

```diff
@@ -0,0 +1,240 @@
+package locust
+
+import (
+	"context"
+	"fmt"
+
+	"go.uber.org/zap"
+	coreV1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
pls do like

```suggestion
	k8sAPIErrors "k8s.io/apimachinery/pkg/api/errors"
```

to avoid mixing stdlib `errors` and k8s errors